conda/README.md

```diff
@@ -0,0 +1,192 @@
+That using conda is very similar. But, there are some important abilities that make conda stand out:
+
+* Very cross platform: Conda is available, and acts very similar on z/OS, Windows, Linux, mac OS, and various Unix.
+                       Packages can state which platforms they support, so it easy to know what packages you can install.
+* Tagging: On z/OS, conda packages can contain tagging information, to avoid issues around the difference between EBCDIC & ASCII
+* Software neutrality: Language-specific package managers are becoming popular, but Conda does not assume the purpose of the package, so you can install most anything.
+* Environments: If desired, every user can have a different set of packages, because conda can install & manage packages in personal folders instead of system ones.
+                A user can even have multiple such environments, and switch between them rapidly to work with different sets of related software without conflict.
+
+
+## Initial Conda Setup
+
+If you have not installed Conda yet, it can be downloaded as an all-in-one package that has no extra dependencies, known as "miniconda".
+For Linux, Unix, and Windows, this can be downloaded at https://docs.conda.io/en/latest/miniconda.html
+For z/OS, Conda can be downloaded from Rocket Software at https://www.rocketsoftware.com/zos-open-source
+
+Conda will prompt during the install for certain setup options, 
+And ultimately you'll want to put some conda initialization content into your startup script
+So that whenever you open your terminal, conda will be ready for your use.
+
+Once you have conda downloaded and installed, you'll want to create your first conda "environment"
+This can be done by providing a path or a nickname
+
+`conda create --prefix PATH`
+`conda create --name ENVIRONMENT`
+
+Either will work, but path helps you better seperate your content from content others use by placing it in a folder that you can have stricter permissions on.
+
+If you need to know more about certain commands, you can use the help command for any.
+
+`conda create --help`
+
+Or, check the official documentation: https://docs.conda.io/en/latest/index.html
+
+Once you have an environment, you should activate it so that the actions you do are on that environment, as opposed to the base one.
+
+`conda activate PATH_OR_NAME`
+
+Conda will detect whether the parameter is a path or a nickname, so this command works for both.
+
+Finally, you can view the conda environment and other information by checking "info"
+`conda info`
+
+## Managing Conda Channels
+
+When downloading a package, such as a Zowe Plugin, the place that you download from is configurable.
+These are called "Channels", but are very similar to "Repositories" seen in other package managers.
+With Conda, you can install from:
+
+* A network channel (Internet or company internal)
+* A local channel (Collection of plugins on your computer)
```

Comment:
Maybe, collection of plugins in your internal conda directory?

I could imagine users getting a Conda package somewhere on the internet and not knowing what to do with it, or why "conda search" can't find it